flipperdevices/flipperzero-firmware
# Fix SubGhz test views erratic behavior on mode switch. New bootloader image.

# What's new

- SubGhz: fix test views erratic behavior on mode switch. 
- Bootloader: new image.

# Verification 

- Compile and upload
- Check DFU mode
- Check Subghz carrier and packet tests.

# Checklist (do not modify)
